libs/types/src/smartcard_auth/dipped_smartcard_auth.ts

```diff
@@ -30,17 +30,19 @@ export interface RemoveCard {
   readonly user: SystemAdministratorUser | ElectionManagerUser;
 }
 
+export type CardSummaryNotReadyStatus = 'no_card' | 'error';
+
 export interface SystemAdministratorLoggedIn {
   readonly status: 'logged_in';
   readonly user: SystemAdministratorUser;
-  readonly card?: CardStorage & CardProgramming;
+  readonly card: CardSummaryNotReadyStatus | (CardStorage & CardProgramming);
   readonly logOut: () => void;
 }
 
 export interface ElectionManagerLoggedIn {
   readonly status: 'logged_in';
   readonly user: ElectionManagerUser;
-  readonly card?: CardStorage;
+  readonly card: CardSummaryNotReadyStatus | CardStorage;
```

Comment:
I thought I saw a comment about why we changed this, but now I can't find it @jonahkagan. I don't understand why there is the possibility of this being anything but undefined (or `no_card`) because if Election Managers don't do smartcard programming then we shouldn't be caring about any cards that are inserted while they're logged in anyway.

Reply:
Why is there a 'card' attribute here at all?

Reply:
The `card` attribute is there to access card storage (aka "long value") (not sure if that's actually used for anything in the product by Election Managers today).

Reply:
(Also, yes, I had commented here but deleted it in favor of my other comment above)